asciidoctorj-core/src/main/java/org/asciidoctor/ast/AbstractBlockImpl.java

```diff
@@ -71,45 +61,25 @@ public Object content() {
 
     @Override
     public Object getContent() {
-        return delegate.content();
-    }
-
-    @Override
-    public String getNodeName() {
-        return delegate.getNodeName();
+        return JavaEmbedUtils.rubyToJava(getProperty("content"));
     }
 
     @Override
     public String convert() {
-        return delegate.convert();
+        return getString("convert");
     }
```

Comment:
The `convert` method should return an Object, even though we've been returning a String. This is related to #234.